### ISSUE TITLE
Screen reader does not read any of the items listed on the Environments overview page

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2235]
Just says "button" for each link/option. This might be fixed by fixing accspot bugs filed earlier, but this open work item can help validate the scenario.

**Comment by huguesv** [comment]
This has been fixed.
